packages/models.py

```diff
@@ -168,8 +168,9 @@ def transfer_to_s3(session, package):
         package.images["s3"] = {}
 
     for idx, image in enumerate(package.images["gdrive"]):
-        req = get_file(session, image["id"], download=True)
-        max_size = (1024, 1024)
+        if image.last_fetched_date < package.last_fetched_date
+            req = get_file(session, image["id"], download=True)
+            max_size = (1024, 1024)
 
```

Comment:
have you tested this? there are two problems: the `last_fetched_date` is probably in a map (and likely a string/timestamp int and not a datetime object) and it probably wouldn't just work like this. also in the else case this would throw an exception if you follow the code